app/views/feedback/feedback.scala.html

```diff
@@ -1,26 +1,30 @@
 @*
- * Copyright 2021 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
 
 @import config.{FormPartialProvider, FrontendAppConfig}
-@import views.html.templates.main_template
+@import views.html.templates.MainTemplate
 
 @import scala.concurrent.ExecutionContext
 
+@this(
+    mainTemplate: MainTemplate
+)
+
 @(partialUrl: String, formBody: Option[Html])(implicit request: Request[_], ec: ExecutionContext, formPartialProvider: FormPartialProvider, messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
EC is not being used could you remove that and change file name to Capital 'F' Feedback?